contrib/confluentinc/confluent-kafka-go/kafka.v2/example_test.go

```diff
@@ -0,0 +1,82 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package kafka_test
+
+import (
+	"fmt"
+
+	"github.com/confluentinc/confluent-kafka-go/v2/kafka"
+
+	kafkatrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/confluentinc/confluent-kafka-go/kafka.v2"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
```

Comment:
Nit:
`"github.com/confluentinc/confluent-kafka-go/v2/kafka"` should come last here.

Stdlib
<line>
Internal (same repo)
<line>
External

Reply:
ACK - rearranged.